vault-github/run.sh

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env bash
+
+BOLD='\033[0;1m'
+VAULT_ADDR=${VAULT_ADDR}
+VERSION=${VERSION}
+AUTH_PATH=${AUTH_PATH}
+GITHUB_EXTERNAL_AUTH_ID=${GITHUB_EXTERNAL_AUTH_ID}
+
+# Check if vault is installed
+if ! command -v vault &>/dev/null; then
```

Comment:
Should we verify it's the correct program and version and install a new one if not?

Reply:
We are already checking for the binary in the path but not the version. I can add a check for the version, too.